drivers/net/mlx5/mlx5_vdpa.c

```diff
@@ -31,6 +31,8 @@
 
 /** Driver Static values in the absence of device VIRTIO emulation support */
 #define MLX5_VDPA_SW_MAX_VIRTQS_SUPPORTED 1
+#define MLX5_VDPA_LOG_RQ_STRIDE           5
```

Comment:
I tried 4 But FW check told me:
create_rq/rmp: log wq stride for rq linked list must be at least 5
So I guess I can change to wq_type to WQ_CYCLIC (right now it's WQ_LINKED_LIST) or leave it to 5. 
Since we are not using the WQ not sure it's matters.
What do you think?